mojoproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 authors = ["Staszek Pasko <staszek@gmail.com>"]
-channels = ["conda-forge", "https://conda.modular.com/max"]
+channels = ["conda-forge", "https://conda.modular.com/max-nightly"]
```

Comment:
My experiences with nightly have been mixed recently and would prefer this repo to work under the stable API... that said 24.5 has not been very stable as well....

Are there any benefits to nightly I don't know about ?